### DISCUSSION TITLE
(Solved) Today's (220824) 'pacman -Syu' left the system unbootable

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#993] [Fixed Issues]
```
rAudio:~  #  pacman -Syu
:: Synchronizing package databases...
 +R                      6.2 KiB  29.8 KiB/s 00:00 [######################] 100%
 core                  257.4 KiB   342 KiB/s 00:01 [######################] 100%
 extra                   2.8 MiB  5.93 MiB/s 00:00 [######################] 100%
 community               6.1 MiB  8.69 MiB/s 00:01 [######################] 100%
 alarm                  51.1 KiB   365 KiB/s 00:00 [######################] 100%
 aur                     9.8 KiB  71.8 KiB/s 00:00 [######################] 100%
:: Starting full system upgrade...
resolving dependencies...
looking for conflicting packages...

Packages (5) geoip-database-20220823-1  linux-rpi-5.15.61-1
             raspberrypi-bootloader-20220823-1  sdl2-2.24.0-1
             x264-3:0.164.r3095.baee400-4

Total Download Size:   31.78 MiB
Total Installed Size:  74.28 MiB
Net Upgrade Size:       0.37 MiB

:: Proceed with installation? [Y/n]
:: Retrieving packages...
 linux-rpi-5.15.6...    24.8 MiB  8.28 MiB/s 00:03 [######################] 100%
 raspberrypi-boot...     4.3 MiB  3.24 MiB/s 00:01 [######################] 100%
 geoip-database-2...  1411.3 KiB  1713 KiB/s 00:01 [######################] 100%
 sdl2-2.24.0-1-aa...   817.6 KiB  1017 KiB/s 00:01 [######################] 100%
 x264-3:0.164.r30...   534.4 KiB   759 KiB/s 00:01 [######################] 100%
 Total (5/5)            31.8 MiB  4.71 MiB/s 00:07 [######################] 100%
(5/5) checking keys in keyring                     [######################] 100%
(5/5) checking package integrity                   [######################] 100%
(5/5) loading package files                        [######################] 100%
(5/5) checking for file conflicts                  [######################] 100%
(5/5) checking available disk space                [######################] 100%
:: Processing package changes...
(1/5) upgrading geoip-database                     [######################] 100%
(2/5) upgrading linux-rpi                          [######################] 100%
warning: directory permissions differ on /etc/
filesystem: 775  package: 755
(3/5) upgrading raspberrypi-bootloader             [######################] 100%
(4/5) upgrading sdl2                               [######################] 100%
(5/5) upgrading x264                               [######################] 100%
:: Running post-transaction hooks...
(1/3) Arming ConditionNeedsUpdate...
(2/3) Updating module dependencies...
(3/3) Updating linux-rpi module dependencies...
rAudio:~  #

```

**Comment by klassikmann** [comment]
Additional info: I found a corresponding topic in the archlinuxarm forum, but without a solution up to now:
https://archlinuxarm.org/forum/viewtopic.php?f=65&t=16149

**Comment by R101G** [comment]
Thanks for the warning @klassikmann !

**Comment by cmh714** [comment]
FWIW, this is my update from yesterday, and I am working....

rAudio:~  #  uname -a
Linux rAudio 5.15.56-4-rpi-ARCH #1 SMP PREEMPT Mon Aug 22 04:04:14 MDT 2022 aarch64 GNU/Linux
rAudio:~  #

**Reply by klassikmann** [reply]
> FWIW, this is my update from yesterday, and I am working....
> 
> rAudio:~ # uname -a Linux rAudio 5.15.56-4-rpi-ARCH #1 SMP PREEMPT Mon Aug 22 04:04:14 MDT 2022 aarch64 GNU/Linux rAudio:~ #

@cmh714 
5.15.61 is the problem maker.

**Comment by klassikmann** [comment]
Now, there are two threads regarding 5.15.61:

1. https://archlinuxarm.org/forum/viewtopic.php?f=65&t=16149 (mentioned above):
The problem applies to eth0 only, not to wifi.

2. https://archlinuxarm.org/forum/viewtopic.php?f=65&t=16152:
The culprit is the bootloader not the kernel.

btw.: The brand-new 5.15.61-2 is not solving the problem.

**Comment by klassikmann** [comment]
News ("help is on the way"?):

Quote from https://lore.kernel.org/all/20220825042106.GA16015@wunner.de/:
> The offending commit was a new feature introduced in v5.19,
> it deliberately wasn't tagged for stable.  Aggressively
> backporting such new features always carries a risk of
> regressions.

Hereupon a maintainer of raspberry/linux decided:
> The back-ports to 5.15 and 5.18 have been reverted.

I'll test the new kernel after the change trickled downstream to ArchLinuxARM.

**Comment by klassikmann** [comment]
> I'll test the new kernel after the change trickled downstream to ArchLinuxARM.

New packages arrived for the suspicious componentes: linux-rpi-5.15.61-3 + raspberrypi-bootloader-20220826-1
'pacman -Syu' + reboot were successful.
